sauces/marinara-sauce.md

```diff
@@ -0,0 +1,35 @@
+# Marinara sauce
+
+This is your nonna's recipe with anchovies and all. This recipe will make a lot of sauce that can freeze easily.
+
+
+## Ingredients
+
+- 4 tbps. olive oil
+- 2 large white onions, chopped
+- 4 cloves of garlic, minced
+- Red pepper flakes to taste to add heat
+- 1 tube of tomato paste
+- 1 tin of anchovies in olive oil, drained
+- 1 large can (28oz) of whole peeled San Marzano tomatoes
+- 1 large can (28oz) of crushed tomatoes
+- 1 tbsp. dried oregano
+- Salt and black pepper to taste
+
+## Instructions
+
+1. **Prep:** Chop onions, mince garlic, drain anchovies from the packaging oil.
+
+2. Heat up olive oil in a large pot, then add the onions to sauté for about 6-8 minutes until they are transluscent.
+
+3. Add minced garlic and red pepper flakes. Stir frequently for 1 minute to make sure the garlic does not burn.
+
+4. Once garlic is fragant, add tomato paste, anchoves, and a couple of pinches of salt and a pinch black pepper. Stir until anchovies dissolve. Let mixture cook until tomato paste darkens for about 5-7 minutes.
+
+5. Dump entire contents of whole and crushed tomatoes into the pot and add oregano. Stir everything well while crushing the whole tomatoes with stirring spoon. Watch out for bubbling and splattering.
+
+6. Once contents start to bubble, lower heat to low and cover partially. Let sauce simmer for about 2 hours. Taste it half way, salt to taste, and stir. 
+
+7. After the sauce has reduced to the desired consistency. It is ready to enjoy with your favorite long pasta.
+
+8. To store let it cool to room temperature. Store it in mason jar in the fridge for up to 4 days. Otherwise, freeze in smaller portions in freezer bags for up to 3 months. Thaw sauce in the fridge for 24 hours before using.
```

Comment:
Why thaw it rather than just heating it up again?

Reply:
It will have a more consistent temperature throughout and also because you may not need the whole sauce at once so heating it up and cooling it back down would not be ideal.